File: Day-1/Factorial.c
// Factorial by recursion function

#include <stdio.h>

int factorial(int x);

int main()
{
    int num, fact;

    printf("Enter number: ");
    scanf("%d", &num);

    fact = factorial(num);

    printf("Factorial of %d is %d", num, fact);

    return 0;
}

int factorial(int x)
{
    if (x == 0 || x == 1)
    {
        return 1;
    }

    return x * factorial(x - 1);
}

File: Day-11/factorial.java
import java.util.Scanner;

public class factorial{

    static int fact(int n){
        int fact = 1;
        for(int i=1; i<=n ; i++){
            fact = fact *i;
        }
        return fact;        

    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a Number: ");
        int n = sc.nextInt();
        System.out.print("Factorial of "+ n + " is " + fact(n));
    }
}